torch/csrc/jit/runtime/static/impl.h

```diff
@@ -16,6 +16,8 @@ struct TORCH_API StaticModuleOptions {
   bool cleanup_activations{true};
   bool enable_out_variant{true};
   bool optimize_memory{true};
+  bool optimize_output_memory{
+      false}; // to enable MemoryPlanner on output tensors
```

Comment:
maybe comment above this line to avoid the line break?

Reply:
Fixed